remote-access/vnc/README.md

```diff
@@ -1,86 +1,104 @@
 
-## Installing VNC
+## Install VNC
 
-VNC is already installed on the full Raspberry Pi OS image, and can be installed via `Recommended Software` from the `Preferences` menu on other versions.
-
-If you are not using a desktop you can install it from the command line as follows:
+VNC Server is already installed on the standard Raspberry Pi OS image; VNC Viewer can be installed using `Recommended Software` from the `Preferences` menu. You can also install both from the command line as follows:
 
 ```bash
 sudo apt update
 sudo apt install realvnc-vnc-server realvnc-vnc-viewer
 ```
 
-## Enabling VNC Server
 
-You can do this graphically or at the command line.
+## Service mode
+
+In this mode, VNC Server will attach to a display which is enabled on the Raspberry Pi itself: remote access to the graphical desktop will be permitted. Note that some Raspberry Pis do not have a display enabled unless a screen is attached.
+
+If you need to change the resolution of the graphical desktop, from the menu choose `Preferences` and click on `Screen Configuration`. You can also use the command line tool `raspi-config`.
 
-### Enabling VNC Server graphically
+### Enable VNC Server
 
-- On your Raspberry Pi, boot into the graphical desktop.
+#### Using the graphical desktop
 
-- Select **Menu > Preferences > Raspberry Pi Configuration > Interfaces**. 
+On your Raspberry Pi, VNC Server can be enabled as follows:
 
-- Ensure **VNC** is **Enabled**. 
+- Boot into the graphical desktop
+- From the menu, point to `Preferences` and click on `Raspberry Pi Configuration`
+- In the `Raspberry Pi Configuration` tool, click the `Interfaces` tab
+- Find the entry for VNC and ensure that `Enabled` is selected
 
-### Enabling VNC Server at the command line
+#### At the command line
 
 You can enable VNC Server at the command line using [raspi-config](../../configuration/raspi-config.md):
 
 ```bash
 sudo raspi-config
 ```
 
-Now, enable VNC Server by doing the following:
+Within `raspi-config`, enable VNC Server as follows:
 
-- Navigate to **Interfacing Options**.
+- Navigate to `Interfacing Options`
+- Scroll down and choose `VNC`, then  `Yes`
 
-- Scroll down and select **VNC > Yes**.
+The VNC system service is named `vncserver-x11-serviced`.
 
-## Connecting to your Raspberry Pi with VNC Viewer
 
-There are two ways to connect to your Raspberry Pi. You can use either or both, depending on what works best for you.
+## Virtual desktop mode
 
-### Establishing a direct connection
+In this mode, VNC Server will create a graphical desktop in the memory of the Raspberry Pi: this allows you to use the graphical desktop of the Pi without having its display enabled.
 
```

Comment:
Do we need some description of what happens if the remote Pi does not have a connected display and you try to connect with VNC? So people have more indication they need to use virtual mode?

Reply:
I've added a description of what happens in this case - not sure if a more prominent warning is needed - this seems to trip up quite a few folk on the forums.

Reply:
Also, please don't integrate this one yet - I'm going look at the proposed changes in #1628, and I think an image of the VNC desktop icon would be useful so folk know what to look for. I will take the WIP tag off once I've done this.